demos/smartlab_online_demo/python/video_processor_serial.py

```diff
@@ -0,0 +1,177 @@
+        self.cap_front = cv2.VideoCapture(front_video_path)
+
+        while self.cap_top.isOpened() and self.cap_front.isOpened() and self.playing:
+            ret_top, frame_top = self.cap_top.read()  # frame:480 x 640 x 3
+            ret_front, frame_front = self.cap_front.read()
+
+            if ret_top and ret_front:
+                self.buffer_top.append(cv2.cvtColor(frame_top, cv2.COLOR_BGR2RGB))
+                self.buffer_front.append(cv2.cvtColor(frame_front, cv2.COLOR_BGR2RGB))
+                self.frame_counter += 1
+
+                print(self.frame_counter)
+                # ''' The object detection module need to generate detection results(for the current frame) '''
+                # top_det_results, front_det_results = self.detector.inference(
+                #         img_top=frame_top, img_front=frame_front)
+
+                ''' The temporal segmentation module need to self judge and generate segmentation results for all historical frames '''
+                if(args.mode == "multiview"):
+                    top_seg_results, front_seg_results = self.segmentor.inference(
+                            buffer_top=frame_top,
+                            buffer_front=frame_front,
+                            frame_index=self.frame_counter
+                            )
+                elif(args.mode == "mstcn"):
+                    top_seg_results, front_seg_results = self.segmentor.inference(
+                            buffer_top=frame_top,
+                            buffer_front=frame_front,
+                            frame_index=self.frame_counter
+                            )
+
+                # ''' The score evaluation module need to merge the results of the two modules and generate the scores '''
+                # self.state, self.scoring = self.evaluator.inference(
+                #         top_det_results=top_det_results,
+                #         front_det_results=front_det_results,
+                #         top_seg_results=top_seg_results,
+                #         front_seg_results=front_seg_results,
+                #         frame_top=frame_top,
+                #         frame_front=frame_front
+                #         )
+
+                # self.display.display_result(
+                #         frame_top=frame_top,
+                #         frame_front=frame_front,
+                #         front_seg_results=front_seg_results,
+                #         top_seg_results=top_seg_results,
+                #         top_det_results=top_det_results,
+                #         front_det_results=front_det_results,
+                #         scoring=self.scoring,
+                #         state=self.state,
+                #         frame_counter=self.frame_counter)
+
+                key = cv2.waitKey(1) & 0xFF
+                if key == ord('q'):     #press 'q' to exit
+                    break
+            else:
+                print("Finished !")
```

Comment:
```suggestion
```
We've unified logging for most of the demos. Please follow it. You can refer to https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/object_detection_demo

Reply:
thanks for the suggesstion. fixed it.